api/krusty/complexcomposition_test.go

```diff
@@ -459,7 +459,7 @@ resources:
 		t.Fatalf("Expected resource accumulation error")
 	}
 	if !strings.Contains(
-		err.Error(), "already registered id: StatefulSet.v1.apps/my-sts.[noNs]") {
+		err.Error(), `id resid.ResId{Gvk:resid.Gvk{Group:"apps", Version:"v1", Kind:"StatefulSet", isClusterScoped:false}, Name:"my-sts", Namespace:""} exists`) {
```

Comment:
This error message was changed due to every `resources` field in another directory being applied independently.